app/views/docs/functions.phtml

```diff
@@ -2,925 +2,67 @@
-  <li class="margin-bottom-small">Select the <b>Manual</b> tab.</li>
-  <li class="margin-bottom-small">Input the entry point of your function under <b>Entrypoint</b>. For the example above, it would be <code>index.js</code>.</li>
-  <li class="margin-bottom-small">Upload <code> code.tar.gz</code>.</li>
-  <li class="margin-bottom-small">Select <b>Activate deployment after build</b> to use your new function.</li>
-  <li class="margin-bottom-small">Click <b>Create</b> to deploy your function.</li>
-</ol>
-
-
-<h2><a href="/docs/functions#build" id="build">Builds</a></h2>
-<p>Deployments needs to be built before they can be activated. This is automatically done after creating a deployment and the time taken can vary depending on the runtime.</p>
-
-<p>If a build fails for any reason, the deployment's status is set to <span class="tag red">failed</span> and you won't be able to activate it. You can however retry a build if you think it was caused by an external factor using the <b>Retry Button</b> on the Appwrite Dashboard or <a href="/docs/server/functions#functionsRetryBuild">Retry Build</a> endpoint with the <code>buildId</code> from the deployment.</p>
-
-<p>To find more details about a deployment and reasons for its failure, you can use the <a href="/docs/server/functions#functionsGetDeployment">Get Deployment</a> endpoint using the <code>deploymentId</code>.</p>
-
-<p>Deployments that have been built successfully are marked as <span class="tag green">ready</span> and can be activated and executed.</p>
-
-<div class="notice">
-    <h3>Build Times</h3>
-    <p>Compiled runtimes such as Rust and Swift take much longer to build however yield better performance over their interpreted counterparts such as Node.</p>
-</div>
-
-<h2><a href="/docs/functions#execute" id="execute">Execute</a></h2>
-
-<p>Besides setting a schedule or allowing your function to listen to Appwrite's system events, you can also manually execute your cloud functions from your Appwrite console or API.</p>
+<h2><a href="#gettingStarted" id="gettingStarted">Getting Started</a></h2>
+<p>
+    Appwrite Functions let you build anything you can imagine, but this flexibility makes it difficult to know where to start.
+    Start exploring by cloning one of the quick start templates or using a template with pre-built integration to quickly implement features.
+</p>
 
 <?php
 $image = new View(__DIR__.'/../general/image.phtml');
     echo $image
-        ->setParam('srcLight', '/images-ee/docs/functions-light.png')
-        ->setParam('srcDark', '/images-ee/docs/functions-dark.png')
-        ->setParam('alt', 'Function settings page.')
-        ->setParam('description', 'Function settings page.')
+        ->setParam('srcLight', '/images-ee/docs/functions-starter-light.png')
+        ->setParam('srcDark', '/images-ee/docs/functions-starter-dark.png')
+        ->setParam('alt', '"Create Function" page.')
+        ->setParam('description', '"Create Function" page.')
```

Comment:
Please update images, it's missing icon, has bad padding.. Preview server should have latest design. Also there is too much blank space at the bottom of image, we can make sure to cut that off.